ops/generate-subgraph.js

```diff
@@ -67,7 +67,10 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
       entities = yamlLoad.entities;
       abis = (yamlLoad.abis || [contract]).map(contractName => {
         let correctedVersion = version;
-        if(contractName == 'TokenTrade' && parseInt(version.slice(-1)[0]) <= 3) {
+        if(contractName === 'Join' && parseInt(version.slice(-1)[0]) <= 5) {
+          correctedVersion = correctedVersion.substring(0, correctedVersion.length -1) + 6
+        }
+        if(contractName === 'TokenTrade' && parseInt(version.slice(-1)[0]) <= 3) {
```

Comment:
why this is needed ?

Reply:
Because these schemes don't exist in older versions, but the controller is using them to index their parameters

Reply:
can we avoid it and make the controller not using them?

Reply:
No, not if we want to index their parameters at lease